config/jobs/kubernetes/sig-release/release-branch-jobs/1.19.yaml

```diff
@@ -1142,7 +1142,7 @@ presubmits:
       - args:
         - --root=/go/src
         - --repo=k8s.io/kubernetes=$(PULL_REFS)
-        - --repo=k8s.io/perf-tests=master
+        - --repo=k8s.io/perf-tests=release-1.19
```

Comment:
This seems like a bug in the jobtemplates that are being forked on new releases - can you please file an issue for this one?

Reply:
I'm not sure about this - Yes, it was generated with master, CI jobs were generated with release-1.19.

However, we actually changed release-1.19 to master (see kubernetes/perf-tests#1384), because perf-tests release-1.19 branch was missing.

I'm more inclined to unifying this on `master` branch, even for CI. WDYT?

Reply:
I disagree. We should attache the jobs to release (and fix the problem that only I can cut the branch in perf-tests).

Reply:
Actually, there is already an ongoing discussion about this in #17097